spec/EmailVerificationToken.spec.js

```diff
@@ -777,4 +777,63 @@ describe("Email Verification Token Expiration: ", () => {
       });
   });
 
+  it('When a user has verified their email, and then change it, the field emailVerified should be false and the new link will be send to the user', done => {
```

Comment:
A suggestion to reword this to be a bit shorter
> emailVerified should be set to false after changing from an already verified email

Reply:
Man, do you speak a lot hahaha.